Update azure-security-disk-encryption.md

deventiwari opened the pull request:
Updated to clarify that LVM is not supported and Linux is ONLY supported on Azure supported gallery images

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jaymathe commented:
I just checked the document and I think it is fine to have the comment in the Azure ML experiment flow diagram to be that way. 
http://azure.microsoft.com/en-us/documentation/articles/machine-learning-r-csharp-normal-distribution/

deventiwari committed:
Update azure-security-disk-encryption.md

Updated to clarify that LVM is not supported and Linux is ONLY supported on Azure supported gallery images
